### ISSUE TITLE
Split stats

### ISSUE THREAD

**Issue by thesimonho** [DallasMcNeil/Block-Keeper#35]
Would it be possible to get stats for splits? For example, Id like to be able to see my mean split 2 (f2l) time as well as the standard deviation for that split alone so I can see how consistent that stage is. Maybe even graph SD over time so I can see if my consistency is improving over time

Given these additional stats, maybe its also worth creating a separate screen for all of these more detailed stats/graphs, instead of trying to cram it into the small boxes? That would give more room for more statistics that may be useful

**Comment by DallasMcNeil** [comment]
I agree, that would be much more helpful. Splits should also have a column too. For now though I don't have the time to implement new features. This would make a great addition in the future.
